### PR TITLE
Add a TTL model for the 74138 3 to 8 decoder

### PR DESCRIPTION
**Pull request by BunchOfBits**
Added a TTL model for the 74138 3 to 8 decoder

### CODE REVIEW

File: src/main/java/com/cburch/logisim/std/ttl/Ttl74138.java [hunk 1]
```diff
@@ -0,0 +1,123 @@
+/*
+ * This file is part of logisim-evolution.
```
Comment (maehne) [inline src/main/java/com/cburch/logisim/std/ttl/Ttl74138.java:2]: Please use consistently "Logisim-evolution".

File: src/main/java/com/cburch/logisim/std/ttl/Ttl74138.java [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.
+ *
+ * Logisim-evolution is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with logisim-evolution. If not, see <http://www.gnu.org/licenses/>.
```
Comment (maehne) [inline src/main/java/com/cburch/logisim/std/ttl/Ttl74138.java:15]: Please use consistently "Logisim-evolution".

**Review by maehne — APPROVED** [review]
Except for two small casing issues, this PR looks good to me.

**Review by maehne — APPROVED** [review]

**Review by maehne — CHANGES_REQUESTED** [review]
Please fix the code style issues reported by GitHub Actions.

**Comment by MarcinOrlowski** [comment]
@BunchOfBits can you please assign an useful title to this PR please?

**Comment by BunchOfBits** [comment]
> @BunchOfBits can you please assign an useful title to this PR please?

Done

**Comment by BunchOfBits** [comment]
> Please fix the code style issues reported by GitHub Actions.

Done

**Review by maehne — APPROVED** [review]

**Comment by btb** [comment]
duplicate of #1462 - either one should work fine.

**Comment by BunchOfBits** [comment]
You are right, I just hadn't scrolled down the list of open PR's that far into the past (june 16th). I don't mind which one is accepted as long as one is accepted. This one has all checks green, while #1462 has still some red checks, so I think it would be easier to accept this one, but either one is fine by me.

**Comment by BFH-ktt1** [comment]
Closing as duplicate of #1462

**Pull request closed without merging**